app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsSurvey.kt

```diff
@@ -0,0 +1,48 @@
+package org.wikipedia.suggestededits
+
+import android.app.Activity
+import android.net.Uri
+import android.widget.TextView
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.settings.Prefs
+import org.wikipedia.util.FeedbackUtil
+import org.wikipedia.util.FeedbackUtil.makeSnackbar
```

Comment:
Please import only `org.wikipedia.util.FeedbackUtil` and use `FeedbackUtil.makeSnackbar` for code consistency.